templates/index.html

```diff
@@ -227,6 +222,11 @@ <h3 class="section-head">Mental State Graph</h3>
                     success: function (data) {
                         data = data[0];
                         console.log(data);
+                        let date = data.timestamp;
```

Comment:
Do you mind moving this block of code you added to be after `if (!data)` check? This makes sure that you don't try to read non existing fields of an empty object.

Reply:
Resolved